### PR TITLE
Fix post image overflow

### PR DESCRIPTION
**Pull request by ddrmv**
The images were bugging me, so here's a quick (inline css) fix with some direct styling. Fixed the linter errors on the way so I could run the full tests. Also, commented out console log statements that were logging the user info, including their password.
Eventually a proper bootstrap way of fixing the images to follow, or even better, limiting the upload size/dimensions or resizing them on upload.
Also, turns out it's really easy to do a new pull request to the team repo from a new fork :)

### CODE REVIEW

**Comment by ddrmv** [comment]
[(s) fix for linter errors and large image overflow on posts page](https://trello.com/c/xTgwPfUK/91-s-fix-for-linter-errors-and-large-image-overflow-on-posts-page)

File: controllers/users.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ const UsersController = {
           isNotOwnProfile: isNotOwnProfile,
         });
       })
-      .catch((err) => {
+      .catch(() => {
```
Comment (44jovi) [inline controllers/users.js:33]: This refactor makes sense

Comment (44jovi) [inline controllers/users.js:33]: This refactor makes sense

File: views/posts/index.hbs [hunk 2]
```diff
@@ -29,7 +29,7 @@
 
                     {{#if this.image.data}}
                       <div><img
-                          class="post-image"
+                          class="post-image img-fluid"
```
Comment (44jovi) [inline views/posts/index.hbs:32]: Very clean way to deal with auto scaling with parent element ✓

File: views/posts/index.hbs [hunk 3]
```diff
@@ -13,7 +13,7 @@
 
                 <div class="col-1">
                   <img
-                    class="author-post-image rounded"
+                    class="author-post-image rounded" style="max-width:72px; max-height:72px"
```
Comment (44jovi) [inline views/posts/index.hbs:16]: This is a nice touch. Maybe in future we use something like "author-image" for the class name for better clarity

**Review by 44jovi — APPROVED** [review]
Good PR!

**Comment by ddrmv** [comment]
[CSS - restrict comment author profile image when commenting on a post](https://trello.com/c/UmZFiaRq/96-css-restrict-comment-author-profile-image-when-commenting-on-a-post)

**Pull request merged**